### PR TITLE
refactor(nms): Drop flow support and generated flow API

### LINKED ISSUE #12995: Remove Flow Support
Remove:
- Flow eslint config + cleanup/consolidate config
- Flow babel config + cleanup/consolidate config
- Flow CI task
- API bindings + generation
- `flow-typed` type stubs
- flow related scripts from package.json
- ...

Remove:
- [ ] server/magma/index.js

### PR DESCRIPTION
**Pull request by thmsschmitt**
## Summary

* Drops flow support and removes all flow related configuration. 
* Updates documentation to replace flow with TypeScript. 
* Removes the flow test from the CI.
* Removes the generated flow API.
* Requires https://github.com/magma/magma/pull/13098.
* Fixes https://github.com/magma/magma/issues/12995.

## Test Plan

```
yarn tsc
yarn eslint
yarn test
```
Also try to introduce an error and check if the tasks fail.

### CODE REVIEW

File: nms/scripts/generateAPIFromSwagger.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (thmsschmitt) [inline nms/scripts/generateAPIFromSwagger.sh:1]: This file was only renamed :(

File: docs/readmes/nms/dev_testing.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ These are the set of tests that you should run before opening a pull request.
 
 #### Eslint
 
-Run `yarn run eslint` to test that there are no linter errors. You can also run `yarn run eslint --fix` to fix automatically fixable linter errors.
+Run `yarn run eslint .` to test that there are no linter errors. You can also run `yarn run eslint --fix .` to fix automatically fixable linter errors.
```
Comment (voisey) [inline docs/readmes/nms/dev_testing.md:21]: Nit: automatically fixable -> automatically-fixable

**Review by voisey — APPROVED** [review]
Docs changes lgtm

File: docs/readmes/nms/dev_testing.md (hunk 2 above)
Comment (Neudrino) [reply docs/readmes/nms/dev_testing.md:21]: Will be fixed in https://github.com/magma/magma/pull/13225/commits/db6a30b0277beef40d51eb5545aa7bf1809c9079.

**Review by Neudrino — APPROVED** [review]

**Review by ShubhamTatvamasi — APPROVED** [review]

**Review by sebathomas — APPROVED** [review]

**Pull request merged**